### ISSUE TITLE
not a git repository warning when building from zip of sources

### ISSUE THREAD

**Issue by brianwaskiewicz** [ntop/PF_RING#311]
I downloaded the 6.6.0 sources zip from github. When building the kernel module, it builds fine, but there is a warning about a git repository not found:

```
fatal: Not a git repository (or any of the parent directories): .git
make -C /lib/modules/4.4.0-21-generic/build SUBDIRS=/home/vagrant/PF_RING-6.6.0-stable/kernel EXTRA_CFLAGS='-I/home/vagrant/PF_RING-6.6.0-stable/kernel ' modules
make[1]: Entering directory '/usr/src/linux-headers-4.4.0-21-generic'
  CC [M]  /home/vagrant/PF_RING-6.6.0-stable/kernel/pf_ring.o
  Building modules, stage 2.
  MODPOST 1 modules
  CC      /home/vagrant/PF_RING-6.6.0-stable/kernel/pf_ring.mod.o
  LD [M]  /home/vagrant/PF_RING-6.6.0-stable/kernel/pf_ring.ko
make[1]: Leaving directory '/usr/src/linux-headers-4.4.0-21-generic'
```

The kernel module still builds correctly (I'm not using DKMS), so it would be nice if the Makefile suppressed this error message (or if some missing information really is needed, provide a way for me to specify it via some other means).

**Comment by cardigliano** [comment]
Hi @brianwaskiewicz, you can safety ignore that error as it does not affect any functionality,
I just pushed a patch to fix it to latest stable 7.0 (6.6 is no longer supported) and dev.
Thank you for reporting.

**Comment by brianwaskiewicz** [comment]
Great, thanks for the quick turnaround.
